packages/postgraphile-core/__tests__/kitchen-sink-schema.sql

```diff
@@ -461,6 +461,54 @@ create function c.return_table_without_grants() returns c.compound_key as $$
   select * from c.compound_key order by person_id_1, person_id_2 limit 1
 $$ language sql stable security definer;
 
+create function c.func_out(out o int) as $$
+  select 42 as o;
+$$ language sql stable;
+
+create function c.func_out_unnamed(out int) as $$
+  select 42;
+$$ language sql stable;
+
+create function c.func_out_setof(out o int) returns setof int as $$
+  select 42 as o
+  union
+  select 43 as o;
+$$ language sql stable;
+
+create function c.func_out_out(out first_out int, out second_out text) as $$
+  select 42 as first_out, 'out'::text as second_out;
+$$ language sql stable;
+
+create function c.func_out_out_unnamed(out int, out text) as $$
+  select 42, 'out'::text;
+$$ language sql stable;
```

Comment:
Good catch! I was just testing this myself:

```sql
begin;
create function foo(out int, out text, out int) returns record as $$ select 1, 'foo', 3; $$ language sql immutable;
select * from foo();
rollback;
```

Reply:
I'd also like to see a test for a mixed named/unnamed, e.g.

```sql
begin; create function foo(out int, out a text, out int) returns record as $$ select 1, 'foo', 3; $$ language sql immutable; select * from foo(); rollback;

 column1 |  a  | column3
---------+-----+---------
       1 | foo |       3
```

Reply:
Added a mixed named/unnamed test in d79a7ff.

Reply:
👍